Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: fetch lint build local-image
+
+export CARGO_ENV_VARS = CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
+
+# Fetches new dependencies from cargo registries
+fetch:
+	$(CARGO_ENV_VARS) cargo fetch --locked 
+
+# Lints rust code via fmt and clippy
+lint:
+	$(CARGO_ENV_VARS) cargo fmt
+	$(CARGO_ENV_VARS) cargo clippy --fix
```

Comment:
Should we pass `--allow-dirty` to allow `cargo clippy --fix` to apply fixes even if there are uncommitted changes caused by running `cargo fmt` before it?